gems/bundler/CVE-2020-36327.yml

```diff
@@ -0,0 +1,28 @@
+---
+gem: bundler
+cve: 2020-36327
+date: 2020-09-30
+url: https://github.com/rubygems/rubygems/issues/3982
+title: |
+  Bundler Gem for Ruby Implicit Dependencies Gemfile Handling Gem Installation
+  Spoofing
+
+description: |
+  Bundler 1.16.0 through 2.2.9 and 2.2.11 through 2.2.16 sometimes chooses a
```

Comment:
Change 2.2.16 to 2.2.17. Looks like 2.2.18 will be the fixed version.

Reply:
Let's add ">= 2.2.18" when that's confirmed.